<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>messageDlg</class>
 <widget class="QDialog" name="messageDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ack.. A Question...</string>
  </property>
  <widget class="QLabel" name="msgLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>431</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
